.github/workflows/pull-request-checks.yaml

```diff
@@ -21,3 +21,22 @@ jobs:
         run: cd build; ninja
       - name: Run CTest
         run: cd build; ctest -V -L CORE .
+  check-vs-2019-build-and-test:
```

Comment:
The name of the action is `build-and-test` because I'm already working on building and running tests on Github Actions under vs2019, but the running tests part is not there yet - however, since the building CBMC is already done, we figured we could publish this part of the work, and get the rest of it merged as it's finished.

Reply:
I don’t think this needs to be called ‘build-and-test’, the other jobs in here are called ‘check’-somethingsomething and I think building _and_ testing should be implied.